README.md

```diff
@@ -1,46 +1,38 @@
 ## How to build
 
-Use `make help` to list possible tasks. But you probably want either
+Use `make help` (`gradlew tasks`) to list possible tasks. But you probably want either
 
-*  `make dist`
+*  `make dist` (`gradlew installDist`)
    which will build a runnable script for you at `cypher-shell/build/install/cypher-shell`
 
 * `make zip`
    which builds a runnable script and packages it up for you as: `out/neo4j-shell.zip`
 
 You can then just run the executable under the `bin/` sub-directory.
 
+## How to run, the fast way
 
-## Development
+This clears any previously known neo4j hosts, starts a throw-away
+instance of neo4j, and connects to it.
 
-### Integration tests
+```sh
+rm -rf ~/.neo4j/known_hosts
+docker run --detach -p 7687:7687 -e NEO4J_AUTH=none neo4j:3.0
+make run
+```
 
-Integration tests are usually skipped when you run the Gradle task `test`.
+## Development
 
-Use Gradle task `integrationTest` to specifically run integrationTests
+### Integration tests
 
-### Pre Requisites
+#### Pre Requisites for running integration tests
 
 Neo4j server with bolt driver configured.
```

Comment:
@spacecowboy : I have made other changes that you had suggested.

The username and password is required to run the integration tests. We need more work on letting the user set the authentication details in tests, but for now we can work with defaults.